### PR TITLE
Show example of local Postgres passwords

### PR DESCRIPTION
**Pull request by bleonard**
## Change description

Give the format for local development to be more clear.

### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [ ] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: cli/templates/.env [hunk 1]
```diff
@@ -41,7 +41,7 @@ REDIS_URL="redis://mock"
 DATABASE_URL="~~SQLITE_DB_PATH~~"
 
 # To use Postgres
-# DATABASE_URL="postgresql://localhost:5432/grouparoo_development"
+# DATABASE_URL="postgresql://username:password@localhost:5432/grouparoo_development"
```
Comment (evantahler) [inline cli/templates/.env]:
```suggestion
# To use Postgres without a username or password
# DATABASE_URL="postgresql://localhost:5432/grouparoo_development"
# To use Postgres with a username and password
# DATABASE_URL="postgresql://username:password@localhost:5432/grouparoo_development"
```

**Review by evantahler — CHANGES_REQUESTED** [review]
I think this is potentially more confusing than it was.
* In our staging apps, we know that people won't have a DB password set if they follow our setup guide
* I do agree with showing an example of having a password in generated `.env` for users, but again if they are using localhost postgres, the default should be no password - maybe just show an example of both?

File: apps/staging-community/.env.example [hunk 2]
```diff
@@ -38,8 +38,10 @@ REDIS_URL="redis://localhost:6379/0"
 # To use sqlite
 # DATABASE_URL="sqlite://grouparoo_development.sqlite"
 
-# To use Postgres
-DATABASE_URL="postgresql://localhost:5432/grouparoo_development"
+# To use Postgres without a username or password
+# DATABASE_URL="postgresql://localhost:5432/grouparoo_development"
```
Comment (evantahler) [inline apps/staging-community/.env.example]: This one should be un-commented

File: apps/staging-enterprise/.env.example [hunk 3]
```diff
@@ -38,8 +38,10 @@ REDIS_URL="redis://localhost:6379/0"
 # To use sqlite
 # DATABASE_URL="sqlite://grouparoo_development.sqlite"
 
-# To use Postgres
-DATABASE_URL="postgresql://localhost:5432/grouparoo_development"
+# To use Postgres without a username or password
+# DATABASE_URL="postgresql://localhost:5432/grouparoo_development"
```
Comment (evantahler) [inline apps/staging-enterprise/.env.example]: This one should be un-commented

**Review by evantahler — CHANGES_REQUESTED** [review]

File: apps/staging-enterprise/.env.example (hunk 3 above)
Comment (bleonard) [reply apps/staging-enterprise/.env.example]: bah. sorry.

**Review by evantahler — APPROVED** [review]

**Pull request merged**